pkg/distro/rhel9/imagetype.go

```diff
@@ -383,16 +380,20 @@ func (t *imageType) checkOptions(bp *blueprint.Blueprint, options distro.ImageOp
 	}
 
 	mountpoints := customizations.GetFilesystems()
-
-	if mountpoints != nil && t.rpmOstree {
-		return warnings, fmt.Errorf("Custom mountpoints are not supported for ostree types")
-	}
-
```

Comment:
I would keep this here and just relax the condition to not apply to `edge-raw-image`. The reason is that doing this check after the default mountpoint policy may result in a situation when the user would get an error related to forbidden mountpoint path for any ostree image type, even one that does not support any FS customizations. That may be confusing and make it look as if the FS customization was supported for the given ostree image type.

Later on, you IMO don't need to check the image type, only if it is ostree-based, because the check for unsupported ostree image types would return from the function before reaching the code.